custom/opm/health_status.py

```diff
@@ -357,14 +411,77 @@ def child_6_months(self):
         return len([c for c in self.all_cases if c.child_age == 6])
 
     @property
-    def child_growth_monitored(self):
-        return len([c for c in self.all_cases if c.child_growth_calculated])
+    def child_growth_monitored_0_3(self):
+        return len([c for c in self.all_cases if c.child_growth_calculated_in_window(3)])
 
     @property
-    def child_mult_3_months(self):
+    def child_0_3_months(self):
         # number of children whose age is a multiple of 3 months
         return len([c for c in self.all_cases
-                    if c.child_age and c.child_age % 3 == 0])
+                    if c.child_age and c.child_age in range(0, 4)])
+
+    @property
+    def child_growth_monitored_4_6(self):
+        return len([c for c in self.all_cases if c.child_growth_calculated_in_window(6)])
+
+    @property
+    def child_4_6_months(self):
+        return len([c for c in self.all_cases
+                    if c.child_age and c.child_age in range(4, 7)])
+
+    @property
```

Comment:
@esoergel these functions are all repititive, and could be made into one. But that involves refactor in the way the indicators are invoked. Do you think I should change this or let it be like this?